GameServerCore/NetInfo/ClientInfo.cs

```diff
@@ -5,10 +5,11 @@ namespace GameServerCore.NetInfo
 {
     public class ClientInfo
     {
-        public long UserId { get; private set; }
-        public int PlayerNo { get; set; }
+        public ulong PlayerId { get; private set; }
+        public uint ClientId { get; set; }
         public bool IsMatchingVersion { get; set; }
         public bool IsDisconnected { get; set; }
+        public bool IsStartedClient { get; set; }
```

Comment:
Personally I would call it `IsClientStarted`, but that's just my personal preference. Looks all good to me at least.